README.md

```diff
@@ -75,19 +75,54 @@ cd sample-pay-service
 
 ```bash
 npm install
-npm install -g ngrok
 ```
 
-3. Set your environment variables
+3. Install Twilio CLI
 
+The Twilio CLI allows you to manage your Twilio resources from your terminal or command prompt. You can use the Twilio CLI below to a provision phone number, configure the webhook, and run ngrok to expose your `localhost` to a public domain so the Twilio webhook can reach the expected endpoint. Alternatively, You can do this in the Twilio Console and by installing `ngrok` separately.
+
+Mac OS X
+```bash
+brew tap twilio/brew && brew install twilio
+```
+
+Windows
+ ```bash
+ npm install twilio-cli -g
+ ```
+
+> See [Twilio CLI Quickstart](https://www.twilio.com/docs/twilio-cli/quickstart) for installation guide for Linux and to learn how to use the Twilio CLI.
+
+4. Log in to Twilio CLI
+```bash
+twilio login
+```
+
+See [Twilio Account Settings](#twilio-account-settings) to locate the necessary environment variables.
+
+5. List phone numbers available for purchase by area code
```

Comment:
There are better commands here:
`twilio phone-numbers:list` and `twilio phone-numbers:buy:...`

Reply:
is `twilio phone-numbers:list` only to show the numbers you own? Looking to list all numbers in area code to purchase. The phone-numbers:buy is new and cool to me. Probably should put that front and center in cli docs.

Reply:
Oh i see it's consolidated

`twilio phone-numbers:buy:local --country-code US --area-code="208"`